addons/popochiu/migration/migration_files/popochiu_migration_1.gd

```diff
@@ -0,0 +1,659 @@
+@tool
+class_name PopochiuMigration1
+extends PopochiuMigration
+## Migrates projects from Alpha to Beta.
+
+const VERSION = 1
+const DESCRIPTION = "Migrate project structure from alpha-x to beta"
+const STEPS = [
+	"Delete [b]res://popochiu/autoloads/[/b].",
+	"Move folders in [b]res://popochiu/[/b] to [b]res://game/[/b]. (pre [i]beta 1[/i])",
+	"Rename folders and files to snake_case. (pre [i]alpha 2[/i])",
+	"Select the default GUI template. (pre [i]beta 1[/i])",
+	"Update paths in [b]res://game/popochiu_data.cfg[/b]. (pre [i]beta 1[/i])",
+	"Rename [b]res://popochiu/[/b] references to [b]res://game/[/b]. (pre [i]beta 1[/i])",
+	"Rename [b]item_xxx[/b] to [b]inventory_item_xxx[/b] for inventory items. (pre [i]beta 1[/i])",
+	"Update [b]PopochiuCharacter[/b]s. (pre [i]beta 1[/i])",
+	"Update external scenes and assign scripts that didn't exist before. (pre [i]beta 1[/i])",
+	"Replace deprecated method calls. (pre [i]beta 1[/i])",
+]
+const RESET_CHILDREN_OWNER = "reset_children_owner"
+const DEFAULT_GUI_TEMPLATE = "SimpleClick"
+const PopochiuGuiTemplatesHelper = preload(
+	"res://addons/popochiu/editor/helpers/popochiu_gui_templates_helper.gd"
+)
+
+var _snake_renamed := []
+
+
+#region Virtual ####################################################################################
+## This is code specific for this migration. This should return [code]true[/code] if the migration
+## is successful. This is called from [method do_migration] which checks to make sure the migration
+## should be done before calling this.
+func _do_migration() -> bool:
+	PopochiuMigrationHelper.is_reload_required = true
```

Comment:
I know this seems to be a moot point, but helper classes should be ideally static (or at least stateless from a public perspective).

What about moving this flag into the class that's named "DoMigrations"?